engine.py

```diff
@@ -1115,9 +1117,7 @@ def _win32_get_proxy_window(self):
                 try:
                     proxy_win_hwnd_found = (
                         self.__tk_photoshopcc.win_32_api.find_windows(
-                            stop_if_found=True,
-                            class_name="Qt5QWindowIcon",
-                            process_id=os.getpid(),
+                            stop_if_found=True, window_text=window_title
```

Comment:
According to tk-framework-adobe documentation, ´find_windows´ might receive multiple parameters to look for all matching windows, if both ´class_name´and ´window_text´ are passed as parameters, the resulting set will be empty since there aren`t any window with both properties, leading to marking the window as "not found" and sending it behind parent window.

Deleting ´class_name´ allow us to find a children window since there is only one filter, fixing the bug.